api/v1alpha08/kogitoserverlessworkflow_flow_types.go

```diff
@@ -1,386 +0,0 @@
-// Copyright 2023 Red Hat, Inc. and/or its affiliates
```

Comment:
This wasn't used ?

Reply:
the purpose of this change is to rely on the sdk-go instead replicate the objects in the operator side.